Makefile

```diff
@@ -14,11 +14,11 @@ test:
 	flake8 --max-line-length=130 ocw
 	flake8 --max-line-length=130 manage.py
 	flake8 --max-line-length=130 cleanup_k8s.py
-	pytest --cov
+	pytest --cov -W ignore::DeprecationWarning
```

Comment:
I am not sure that this is good idea :thinking:  maybe we need to follow warnings and stop using what is deprecated instead ? use of deprecated API is increase of technical debt which you will need to pay anyway one day ...

Reply:
> I am not sure that this is good idea thinking maybe we need to follow warnings and stop using what is deprecated instead ? use of deprecated API is increase of technical debt which you will need to pay anyway one day ...

The deprecated warning comes from Django 4.0.x.

Either we update Django or use this temporary workaround, because pytest refuses to run without it.

Reply:
than we need to update Django !